src/python/bench/benchmark.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+
+function translate_native {
+ sacrebleu -t wmt20 -l en-de --echo src | \
+   spm_encode --model /home/marcinjd/MTMA/source.spm |  ../_skbuild/linux-x86_64-3.8/cmake-build/marian-decoder --quiet \
```

Comment:
Is the path `../_skbuild/linux-...` generic so that it is going to work for most users? If not, consider extracting it to a global variable.